Automattic/jetpack
# CloudFlare Analytics: move feature to mu-wpcom

## Proposed changes:

The feature is only used on WordPress.com sites, it is not needed in the Jetpack plugin.

Since the feature was loaded on all connected Jetpack sites, it also had the inconvenient of making an option call on every frontend call.

### Other information:


## Jetpack product discussion

* pfwV0U-3n-p2

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

Nothing should change on Jetpack sites. You should consequently test this on WordPress.com sites:

1. Go to https://wordpress.com/marketing/traffic/
2. Enable CloudFlare anaylitics and paste in a tracking code.
3. Check the site's frontend; you should see the tracking code added in the footer.
4. Now switch to this branch with the Jetpack plugin and the mu-wpcom plugin; the code should still load.
